fusor-ember-cli/app/templates/review/progress.hbs

```diff
@@ -31,16 +31,18 @@
   </div>
 </div>
 
-{{#cancel-back-next backRouteName='review.installation'
-                    disableBack=false
-                    disableCancel=isStarted}}
-  {{#if deployTaskIsFinished}}
-    {{#link-to 'review.summary' role="button" class='btn btn-primary'}}
-        Next <i class="fa fa-angle-right"></i>
-    {{/link-to}}
-  {{else}}
-    <button class='btn btn-primary' disabled=true>
-        {{deployButtonTitle}} <i class="fa fa-angle-right"></i>
-    </button>
-  {{/if}}
-{{/cancel-back-next}}
+{{#unless deploymentController.model.has_content_error}}
```

Comment:
Having a hard time reading the unless. So the buttons will show IF the deploymentController does NOT have any content errors right? Is that what the unless effectively does? While it might seem unnecessary, even a simple comment above that shows the positive (I think that's the word) logic path would help. i.e. ```// only show the buttons if no content errors```